script/test

```diff
@@ -0,0 +1,21 @@
+# Install cld2-cffi package with an additional CFLAG
+CFLAGS="-Wno-narrowing" pip3 install --quiet cld2-cffi
+
+# Install requirements
+pip3 install --quiet --requirement requirements.txt
+
+# Install and run pylint
+pip3 install --quiet --upgrade pylint 
+python3 -m pylint -f parseable textpipe | tee pylint.out
```

Comment:
Maybe consider flake8?

Reply:
In my experience, pylint is a more detailed than flake8 even though the output may be a bit much at times (but you can easily disable the warnings or conventions). What are your thought on flake8 vs pylint?

Reply:
Also only a slight preference from our side. We like that it actually assigns a score to the code that we can track over time and with PRs. I would propose to keep using `pylint` for now.